styles.css

```diff
@@ -1,122 +1,157 @@
 body {
-    margin: 0;
-    font-family: "Montserrat";
+  margin: 0;
+  font-family: "Montserrat", sans-serif;
 }
 
 /* ---------------------HEADER SECTION--------------- */
 .header {
-    /* border: 0.5em solid black; */
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    margin: 0;
-    color: #FFF;
-    background-color: #46424D;
-    height: 10vh;
+  /* border: 0.5em solid black; */
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  color: #FFF;
+  background-color: #46424D;
+  height: 8.5vh;
+}
+
+.h1 {
+  font-size: 1.35rem;
+  font-weight: 300;
 }
 
 /* ---------------------MAIN SECTION--------------- */
 .main-container {
-    /* border: 0.5em solid black; */
-    display: flex;
-    flex-direction: row;
-    justify-content:space-between;
-    height: 95vh;
-    color: #FFF;
-    background-color: #46424D;
+  /* border: 0.5em solid black; */
+  display: flex;
+  flex-direction: row;
+  justify-content:space-between;
+  height: 91.5vh;
+  color: #FFF;
+  background-color: #46424D;
+}
+
+.h2 {
+  font-size: 1.5rem;
+  font-weight: 300;
+  color: #CBC9CF;
 }
 
 /* ---------------------NEW ACTIVITY SECTION--------------- */
 .new-activity {
```

Comment:
Box shadow is along these lines. Not sure if this the correct location as the shadow isn't real clear to me in the comp.

box-shadow: 0 19px 38px rgba(0,0,0,0.20), 0 15px 12px rgba(0,0,0,0.

Reply:
Done! And adjusted while we were on the call.